data/lists.js

```diff
@@ -3,3 +3,168 @@
 // edit list - used by /list/:id route
 // delete list - used by /list/:id route
 // delete entry from list - used by /list/:id route
+import { users } from "../config/mongoCollections.js";
+import { ObjectId } from "mongodb";
+import * as validation from "../utils/validation.js";
+
+const exportedMethods = {
+  async create(userId, title, movies) {
+    //for validation validate if
+    //userId is valid
+    //title is a string
+    //movies is an array of integers
+    validation.checks(userId);
+    userId = userId.trim();
+    if (!ObjectId.isValid(userId)) throw "invalid object ID";
+    title = title.trim();
+    if (!movies || !Array.isArray(movies))
+      throw "You must provide an array of movies";
+    if (movies.length === 0) throw "You must supply at least one movie";
+    movies.forEach((elem) => {
+      elem = validation.checkNumber(elem, "movie");
+    });
+    const userCollection = await users();
+    const user = await userCollection.findOne({ _id: new ObjectId(userId) });
+    if (user === null) throw "No user with that id";
```

Comment:
Since users data is in develop, you can import the file into here like: `import * as userData from './users.js'` and then replace this line with:
```suggestion
    const user = await usersData.getUserById(userId);
```
and the call will take care of any checks for you